index.html

```diff
@@ -198,13 +199,71 @@ <h2>On behalf of organisations</h2>
 
 			</article>
 
+			<article class='container' lang='de'>
+
+				<p>Wir sind eine Gruppe von Einzelpersonen, die ein erhebliches Interesse an der Entwicklung und Gesundheit des World Wide
+					Web ("das Web") haben, und <strong>wir sind zutiefst besorgt</strong> über <a href='https://ampproject.org'>Accelerated Mobile Pages</a> ("AMP"), ein Google-Projekt, das angeblich darauf abzielt, die Benutzererfahrung des Webs
+					zu verbessern.</p>
+	
+				<p>Tatsächlich hält AMP die Nutzer innerhalb der Google-Domain und leitet den Verkehr von anderen Websites zum Nutzen von
+					Google ab. Mit Milliarden von Nutzern hat dies zur Folge, dass die Dominanz von Google im Web weiter verstärkt wird.</p>
+	
+				<p>Wir erkennen das Problem langsamer Ladezeiten von Webseiten im Vergleich zu alternativen, proprietären Technologien wie <a href='https://instantarticles.fb.com/'>Facebook Instant Articles</a> und <a href='https://developer.apple.com/news-publisher/'>Apple News</a> an. Verlage (vor allem in den Nachrichtenmedien) sind seit langem mit schwierigen Entscheidungen und schlechten
+					Anreizen konfrontiert, die zu Fehlentscheidungen, Kompromissen und letztlich zu schrecklichen Benutzererfahrungen führen.</p>
+	
+				<p>Suchmaschinen sind in der Lage, Einfluss auszuüben, um dieses Problem zu lösen. Google hat sich jedoch dafür entschieden,
+					eine Premium-Position an der Spitze ihrer Suchergebnisse (für Artikel) und ein "Blitz"-Symbol (für alle Arten von Inhalten)
+					zu schaffen, die nur für Publisher zugänglich sind, die eine von Google gesteuerte Technologie verwenden, die von Google
+					von ihrer Infrastruktur aus über eine Google-URL bedient wird und innerhalb einer von Google kontrollierten Benutzererfahrung
+					platziert wird. (<a href='https://developers.google.com/search/docs/guides/about-amp'>Quelle</a>)</p>
+	
+				<p>Das AMP-Format ist nicht an sich ein Problem, aber zwei Aspekte seiner Implementierung stärken die Position von Google
+					als de facto Standardplattform für Inhalte, da Google versucht, die Akzeptanz von AMP bei den Content-Erstellern zu erhöhen:</p>
+	
+				<ol>
+					<li>Inhalte, die sich für AMP und das damit verbundene Hosting innerhalb der Google-Domain entscheiden, erhalten eine bevorzugte
```

Comment:
I am not sure if you can say "Inhalte, die sich für AMP [...] entscheiden", since a "content" is unable to make a "decision". I am not sure if you can say this in English, but it doesn't sound good in German. But then, this is just opinion based.

Reply:
I agree. Better would be: "Publizisten, die sich für AMP und das damit verbundene Hosting ihrer Inhalte innerhalb der Google-Domain entscheiden,..."